packages/sanes-chrome-extension/src/logic/persona/personaManager.ts

```diff
@@ -8,6 +8,23 @@ export class PersonaManager {
    */
   public static async create(fixedMnemonic?: string): Promise<Persona> {
     const newInstance = await Persona.create(fixedMnemonic);
+
+    const server = newInstance.startSigningServer();
```

Comment:
This should live in the chrome extension's background script, and data (if needed) should be passed to the extension via redux.

Find the `wrapStore(makeStore());` line before. Then doing store.dispatch(oneAction), you can have reducers & listeners in the extension for handling everything. This is the natural way of doing it.

Anyway, if you have found another way to send data from background script to extension, and being able to store it somehow in case of multiple extension open and closes let me know, we could remove redux at all (when I researched this, I did not find a proper way).

Reply:
The `MultiChainSigner` instance cannot be passed between UI process and background script as it is not copyable. We can either move the entire Persona class or nothing.

The advantage of keeping it in the UI process is that the connections to the blockchains are re-established as soon as the extension is re-opened. If we move it to the background script, a closed WebSocket connection will break the functionality of the extension as long as the browser is open. In https://github.com/iov-one/iov-core/issues/229 we want to build automatic retry and reconnect into IOV-Core. But we are quite far from getting this right. In the meantime we should use a new `MultiChainSigner` instance with new chain connections for every interaction.